registry/txt.go

```diff
@@ -371,20 +396,24 @@ func (pr affixNameMapper) isSuffix() bool {
 	return len(pr.prefix) == 0 && len(pr.suffix) > 0
 }
 
-func (pr affixNameMapper) toEndpointName(txtDNSName string) (endpointName string, isAAAA bool) {
-	lowerDNSName, recordType := extractRecordType(strings.ToLower(txtDNSName))
+func (pr affixNameMapper) toEndpointName(txtDNSName string) (endpointName string, recordType string) {
```

Comment:
same as in https://github.com/kubernetes-sigs/external-dns/pull/3724/files#r1290447277 , I would be in favor of dropping the "named return" in case we do not use it.